src/components/commons/inputs/textFields/TextField.style.ts

```diff
@@ -0,0 +1,64 @@
+import { Generators } from "@styles/generator";
+import styled from "styled-components";
+import { IconTextfiedlDelete } from "@assets/svgs";
+
+export const TextFieldLayout = styled.section<{ narrow?: false | true }>`
+  position: relative;
+  width: ${({ narrow }) => (narrow ? "13.6rem" : "32.7rem")};
+`;
+export const TextFieldWrapper = styled.article`
+  ${Generators.flexGenerator("row", "center", "center")}
+`;
+export const TextFieldInput = styled.input`
```

Comment:
p4) 여기 가독성 좋게 한 칸씩 띄어서 써주면 좋을 것 같아요!
```suggestion
export const TextFieldLayout = styled.section<{ narrow?: false | true }>`
  position: relative;
  width: ${({ narrow }) => (narrow ? "13.6rem" : "32.7rem")};
`;

export const TextFieldWrapper = styled.article`
  ${Generators.flexGenerator("row", "center", "center")}
`;

export const TextFieldInput = styled.input`
```